src/__tests__/Icon.js

```diff
@@ -22,9 +22,8 @@ describe('Icon', () => {
     })
   })
 
-
   test('returns false for non-existing icons', () => {
-    const icon = renderer.create(<Icon name='nope' />).toJSON()
+    const icon = renderer.create(<Icon name="nope" />).toJSON()
```

Comment:
huh. Prettier prefers double quotes @jxnblk?

Reply:
👎

Reply:
It does single quotes for plain JS, and double quotes in JSX – I'm not going to fight the machines on this one